client/app/scripts/superdesk-users/users.js

```diff
@@ -252,13 +253,26 @@
                 };
 
             if (params.q) {
-                criteria.where = JSON.stringify({
+                var query = {
```

Comment:
There is a UserListService in the same file on line 90. Maybe we should use that service here. I have another PR (https://github.com/superdesk/superdesk/pull/404) where I supposed to do the same actually. What do you think?